automated/linux/iperf/iperf-server.yaml

```diff
@@ -0,0 +1,38 @@
+metadata:
+    name: iperf
+    format: "Lava-Test-Shell Test Definition 1.0"
+    description: "iperf is a tool for active measurements of the maximum
+                  achievable bandwidth on IP networks."
+    maintainer:
+        - chase.qi@linaro.org
+    os:
+        - debian
+        - ubuntu
+        - fedora
+        - centos
+    scope:
+        - performance
+    environment:
+        - lava-test-shell
+    devices:
+        - hi6220-hikey
+        - apq8016-sbc
+        - mustang
+        - moonshot
+        - thunderX
+        - d03
+        - d05
+        - soca9
+        - rzn1d
+
+params:
+    SKIP_INSTALL: "false"
+    SERVER_ETHERNET_DEVICE: "eth0"
+
+run:
+    steps:
+        - cd ./automated/linux/iperf/
+        - lava-send server-ready ipaddr=$(lava-echo-ipv4 ${SERVER_ETHERNET_DEVICE})
```

Comment:
maybe this should be split into to lines. In case lava-echo-ipv4 returns empty string we should abort with lava-test-raise.

Reply:
I missed this comment, I'll address it now...

Reply:
Ugh, I split it and get this error:
/lava-1972096/0/tests/0_iperf3-test/run.sh: line 24: unexpected EOF while looking for matching `\"'
I guess I need to run it through "tr" or something to deal with the null.